components/bl-mapbox-component/README.md

```diff
@@ -10,24 +10,27 @@ The component covers basic features of the library and allows you in a couple cl
+|-----------------------|---------------------------------------------------------------------------------------------------------------------------------|--------------------------------------|----------------|--------------|------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Markers               | *JSON*                                                                                                                          |                                      | Markers Logic  | YES          | YES        | Specifies an array of markers to display on the map. Watch [Codeless Examples](#codeless-examples). Signature of marker: `{color, description, coordinates: {lat, lng}}`.                               |
+| Polygons              | *JSON*                                                                                                                          |                                      | Polygons Logic | YES          | YES        | Specifies an array of polygons to display on the map. Watch [Codeless Examples](#codeless-examples). Signature of polygon: `{color, description, opacity, polygon: {boundary: points: [ {lat, lng}]}}`. |
+| Polygon Outline       | *Checkbox*                                                                                                                      | `false`                              |                | NO           | YES        | Controls visibility of polygon outline.                                                                                                                                                                 |
+| Polygon Outline Color | *Color*                                                                                                                         | '#000000'                            |                | NO           | YES        | Sets the polygon outline color.                                                                                                                                                                         |
+| Polygon Outline Width | *Number*                                                                                                                        | 2                                    |                | NO           | YES        | Setts the polygon outline width.                                                                                                                                                                        |
```

Comment:
Could it be added to polygon signature? In that case we have an ability to create different polygons.